INSTALL.md

```diff
@@ -32,7 +32,7 @@ Now you can install the python dependencies by calling:
     $ sudo pip install -e .
 
 *Note: You might have to install the additional Ubuntu/Debian packages
-`libpq-dev`, `python-dev` and `g++` for the `psycopg2` dependency.*
+`libpq-dev`, `python-dev` and `g++` for the `psycopg2` dependency. Furthermore the XCSoar tools might require additional dependecies (see XCSoar tools section)*
```

Comment:
- "dependecies" is missing an "n"
- can you break the lines at 80 characters like the other are too?

Reply:
will do, thanks